rootfs/scheduler/__init__.py

```diff
@@ -176,13 +177,14 @@ def _get_deploy_batches(self, steps, desired):
 
         return batches
 
-    def _update_application_service(self, namespace, name, app_type, port, routable=False):
+    def _update_application_service(self, namespace, name, app_type, port, routable=False, maintenance=False):  # noqa
```

Comment:
currently there is only one annotation and one label which is updated through controller.